src/renderers/webgl/WebGLTextures.js

```diff
@@ -1040,11 +1037,12 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 		const forceUpload = initTexture( textureProperties, texture );
 		const source = texture.source;
 
-		state.activeTexture( _gl.TEXTURE0 + slot );
-		state.bindTexture( _gl.TEXTURE_CUBE_MAP, textureProperties.__webglTexture );
+		state.bindTextureToSlot( _gl.TEXTURE0 + slot, _gl.TEXTURE_CUBE_MAP, textureProperties.__webglTexture );
 
 		if ( source.version !== source.__currentVersion || forceUpload === true ) {
 
+			state.activeTexture( _gl.TEXTURE0 + slot );
```

Comment:
Why is it necessary to call `activeTexture()` here when `bindTextureToSlot()` was executed right before?

Reply:
It's possible that the version was bumped for an already bound texture.  If the texture was already bound to a slot,  BindTextureToSlot exits without calling activeTexture.  Fortunately activeTexture on the three.js side already caches which slot is active, so if we've bound the texture and then drop into here, we simply check the cache and exit out without duplicating the gl call.

Reply:
Also, it's not theoretical: we had a case where we were preloading bone textures and they weren't updating properly before I added this.